src/lib/misc.ts

```diff
@@ -193,26 +193,26 @@ export function getOutcomeTranslation(outcome: string) {
     return outcome;
 }
 
-export function getGameResultText(game) {
+export function getGameResultText(outcome: string, white_lost: boolean, black_lost: boolean): string {
     /* SGFs will encode the full result in the outcome */
-    if (/[+]/.test(game.outcome)) {
-        return game.outcome;
+    if (/[+]/.test(outcome)) {
+        return outcome;
     }
 
     let winner = "";
     let result = "";
-    if (game.white_lost && game.black_lost) {
+    if (white_lost && black_lost) {
         winner = _("Tie");
-    } else if (game.white_lost) {
+    } else if (white_lost) {
         winner = _("B");
-    } else if (game.black_lost) {
+    } else if (black_lost) {
         winner = _("W");
     } else {
         winner = _("Tie");
     }
 
-    game.outcome = game.outcome.replace(" points", "");
```

Comment:
this seems to be the main difference between my implementation and the initial one. The initial code changes the `outcome` text that is in the `game` object. It doesn't seem to have any effect... but it's hard to be 100% sure... comments?

Reply:
Really hard to tell.  But I also wouldn't expect a function named `getGameResultText(game)` to modify `game`, so this update is probably a move in the right direction.

Reply:
Yeah 100% agree, no idea why I assigned that here, it most certainly shouldn't. Thanks for catching and and fixing that @maurelio1234 !